timers.js

```diff
@@ -1,4 +1,31 @@
-
-
-
-
+// Step 1.
+var resetButton = document.querySelector("#reset");
+var startButton = document.querySelector("#start") ;
+var pauseButton = document.querySelector("#pause");
+
+// Step 2. & Step 5,6,7
+
+resetButton.addEventListener("click");
+
+startButton.addEventListener("click", function() {
+	watchTimer = setInterval(updateTime, 1000);

```

Comment:
Rather than instantiate global variables inside of your event listener, why not use that `timerId` variable you created in Step 3 to contain your `setInterval`s?